plugin/federated/README.md

```diff
@@ -5,13 +5,20 @@ This folder contains the plugin for federated learning. Follow these steps to bu
 
 Install gRPC
 ------------
+We highly recommend installing gRPC in a local environment, such as a Conda environment,
```

Comment:
Can we just refer to the gRPC document instead of writing our own?

Reply:
Done